analyses/malawi/flooding_trigger/mwi-historical-baseline.R

```diff
@@ -0,0 +1,64 @@
+library(dplyr)
+library(sf)
+library(lubridate)
+library(readxl)
+library(yaml)
+library(tmap)
+
+# Get overall config params and read in data
+data_dir <- Sys.getenv("AA_DATA_DIR")
+data_private_dir <- Sys.getenv('AA_DATA_PRIVATE_DIR')
+config <- read_yaml('../../src/malawi/config.yml')
+shp_adm2 <- st_read(paste0(data_dir, '/raw/malawi/Shapefiles/', config$path_admin2_shp))
+shp_adm3 <- st_read(paste0(data_dir, '/raw/malawi/Shapefiles/', config$path_admin3_shp))
+df_mvac_floods <- read.csv(paste0(data_private_dir, "/processed/malawi/mvac_dodma_flood_info.csv"))
```

Comment:
Only tried to run the file without review
the paths here are outdated. Not working anymore with the `AA_DATA_PRIVATE_DIR` env variable, and the shapefiles paths changed